modules/earthengine-layers/src/ee-api.js

```diff
@@ -29,7 +33,7 @@ export default class EEApi {
     });
   }
 
-  // TODO keep? - From github test code
+  // Set short-lived Access Token directly
```

Comment:
Do we need the `setCloudAPIEnabled` stanza below? I think a copied that from the tests inside the earthengine API, so may be related to running unit tests without cloud access (something we may want to do) but may not be needed for normal use.

Reply:
I don't know what it does. I need to research that.

Reply:
I think it adds a scope when authenticating to access Google Cloud Platform.
https://github.com/google/earthengine-api/blob/aca5f8eba9001182d861bdd70e154873700f30ca/javascript/src/apiclient.js#L1496-L1500

I don't know when that's necessary, but probably not necessary for most use cases.

Reply:
I recommend removing it for now since it nothing indicates that we need it outside of tests.